### ISSUE TITLE
[Feat] Postal Codes 

### ISSUE THREAD

**Issue by JorgeMiguelGomes** [vostpt/api#24]
Include and cross-reference postal codes

**Comment by quetzyg** [comment]
Unless someone knows of a better source, I'll be using [this](https://github.com/centraldedados/codigos_postais/blob/master/data/codigos_postais.csv).

**Comment by josepostiga** [comment]
Is this issue still relevant? If so, care to give a little more detail about to what cross-reference with?

**Comment by quetzyg** [comment]
Well, the issue is relevant, but the problem is that there's no easy way to map a postal code to a parish. The most you can do in an automated fashion is map a postal code to a council. You could try doing it by hand, but there are 300K+ entries, so...

**Comment by joaoleveiga** [comment]
Hello! I'm new around here :) What exactly is the issue/enhancement?

Is it to find a way to automatically map parishes to postal codes? What keys are expected for the mapping (besides the postal code)? NUTS IDs, some other ID, ...?

Just trying to understand if I can help in any way.

**Comment by josepostiga** [comment]
@joaoleveiga We can't find a way to programmatically associate postal codes to parishes, directly. The only relation we can make is between counties and postal codes.

We only need the relation between the postal codes and the parishes. Unless you really need it, there's no real justification to use other keys than those, as far as I can tell.

**Comment by joaoleveiga** [comment]
@josepostiga @quetzyg Have you tried looking at this: https://www.geonames.org/postalcode-search.html?q=&country=PT ?

**Comment by quetzyg** [comment]
Nope, but I doubt the DICOFRE code is available there to allow us to map stuff.
